src/linux/common/helpers/azure-auto-recover/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+        "C_Cpp.errorSquiggles": "Disabled"
```

Comment:
to be removed

Reply:
The file has been removed thanks for pointing out about it